### PR TITLE
PD-188780: Add backstage config

### PR DESCRIPTION
**Pull request by Artefall**
## Github Issue #

[PD-188780](https://github.com/bazaarvoice/emodb/issues/PD-188780)

## What Are We Doing Here?

Adding backstage config

## How to Test and Verify

1. Create new entity in backstage via catalog-info.yaml config file

### Level 

Low 

### Required Testing

Manual

### Risk Summary

Risk of invalid config file

## Code Review Checklist

- [x] Tests are included. If not, make sure you leave us a line or two for the reason.

- [x] Pulled down the PR and performed verification of at least being able to
build and run.

- [x] Well documented, including updates to any necessary markdown files. When
we inevitably come back to this code it will only take hours to figure out, not
days.

- [x] Consistent/Clear/Thoughtful? We are better with this code. We also aren't
a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.

- [x] PR has a valid summary, and a good description.

### CODE REVIEW

File: catalog-config.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  name: maven-cassandra-plugin
+  description: Cassandra Maven Plugin
+  annotations:
+    backstage.io/techdocs-ref: 'url:https://github.com/bazaarvoice/maven-cassandra-plugin/blob/master/README.md'
+  links:
+    - url: https://github.com/bazaarvoice/maven-cassandra-plugin/blob/master/README.md
+      title: Github docs
+      icon: docs
+  tags:
+    - prr
```
Comment (mike-unitskyi) [inline catalog-config.yaml:13]: the component is related to emodb, not prr

File: catalog-config.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    - java
+    - cassandra
+spec:
+  type: library
```
Comment (mike-unitskyi) [inline catalog-config.yaml:17]:
```suggestion
  type: maven-plugin
```

File: catalog-config.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  system: prr
```
Comment (mike-unitskyi) [inline catalog-config.yaml:18]:
```suggestion
  system: emodb
```

**Review by mike-unitskyi — CHANGES_REQUESTED** [review]

File: catalog-config.yaml (hunk 1 above)
Comment (Artefall) [reply catalog-config.yaml]: Fix

**Pull request closed without merging**